samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -616,6 +616,9 @@
             </li>
           </ul>
           <h:outputLabel id="selTimeAssessHelpBlock" styleClass="help-block info-text small" value="#{assessmentSettingsMessages.assessment_timed_info}" />
+          <h:panelGroup rendered="#{assessmentSettings.assessment.hasMultipleTimers() > 1}" layout="block" id="multipleTimersInfo" styleClass="sak-banner-warn">
+            <h:outputText value="#{assessmentSettingsMessages.multiple_timers_detected}" />
```

Comment:
Will this be detected and shown, when the instructor is adding multiple timers at once, without saving or refreshing in-between?

Reply:
Lets move this to the confirmation screen.

Reply:
Duplicated in confirmation publish screen. Message will be shown in both screens